src/ORM/Table.php

```diff
@@ -2105,9 +2105,14 @@ protected function _insert(EntityInterface $entity, array $data): EntityInterfac
             $entity->set($filteredKeys, ['guard' => false]);
             $schema = $this->getSchema();
             $driver = $this->getConnection()->getDriver();
+            $returning = $statement->fetchAssoc();
```

Comment:
This is better than using `PDO::lastInsertId()`?

Reply:
For drivers that support RETURNING and similar syntax, the query returns a row with the generated IDs. PDO::lastInsertId() either returns the last ID for a table or for a sequence. For example, you can call lastInsertId() for a table on postgresql. You must fetch the row retuned from INSERT.

Reply:
Reverted.